src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int TYPE_INDEX = 0;
+    private static final int SUM_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readFile(fromFileName);
+        String report = createReport(fileContent);
+        writeReportToFile(toFileName, report);
+    }
+
+    private String readFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            return builder.toString();
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+    }
+
+    private String createReport(String fileContent) {
+        String[] lines = fileContent.split(System.lineSeparator());
+        int totalSupply = 0;
+        int totalBuy = 0;
+        for (String line : lines) {
+            String[] words = line.split(",");
```

Comment:
that's not exactly words. Of course `supply` and `buy` is, but numbers aren't
```suggestion
            String[] lineData = line.split(",");
```